src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -32,36 +33,50 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
   } = props;
 
   const { width, height } = useThree((state) => state.size);
+  const camera = useThree((state) => state.camera);
   const gl = useThree((state) => state.gl);
   const { parentElement } = gl.domElement;
-  const camera = useThree((state) => state.camera);
 
   // Container `div` for ReactDOM to render into, appended next to R3F's `canvas`
-  const [el] = useState(() => document.createElement('div'));
-  // Placeholder R3F group, the position of which is tracked and forwarded to `el` when `followCamera` is enabled
+  const [el] = useState(() => {
+    const elem = document.createElement('div');
+
+    // Take out of flow and let pointer events pass through to canvas
+    elem.style.cssText = `position: absolute; top: 0; left: 0; pointer-events: none;`;
```

Comment:
Isn't it possible to set `overflow` here ?

Reply:
The `overflow` depends on a prop, so it needs to be updated dynamically. Note that since it's set in a `useLayoutEffect`, there's no chance of seeing a glitch on first render (i.e. the browser will always paint the DOM _after_ applying the correct overflow).